http/org.wso2.carbon.transport.http.netty/src/test/java/org/wso2/carbon/transport/http/netty/util/client/websocket/WebSocketClientHandler.java

```diff
@@ -108,9 +111,15 @@ public void channelRead0(ChannelHandlerContext ctx, Object msg) throws Exception
             BinaryWebSocketFrame binaryFrame = (BinaryWebSocketFrame) frame;
             bufferReceived = binaryFrame.content().nioBuffer();
             logger.debug("WebSocket Client received  binary message: " + bufferReceived.toString());
```

Comment:
better check whether debug is enabled when adding debug logs with string concatinavtion